applicationset/generators/pull_request.go

```diff
@@ -96,6 +97,7 @@ func (g *PullRequestGenerator) GenerateParams(appSetGenerator *argoprojiov1alpha
 			"branch_slug":    slug.Make(pull.Branch),
 			"head_sha":       pull.HeadSHA,
 			"head_short_sha": pull.HeadSHA[:shortSHALength],
+			"labels":         strings.Join(pull.Labels, ","),
```

Comment:
Looks like this produces a flat, comma-delimited string, regardless of whether you're using go templating.

Can this be made to produce a proper array of strings, so that go templates are easier to write? I'd be okay making this param only available for go-template appsets, because I expect we'll deprecate non-gotemplated appsets within a few releases.

Reply:
I have changed it to product an array. The 'labels' will be available only for go-template appsets.